ReactNativeClient/lib/components/screens/config.js

```diff
@@ -253,6 +254,7 @@ class ConfigScreenComponent extends BaseScreenComponent {
 
 	renderHeader(key, title) {
 		const theme = themeStyle(this.props.theme);
+		theme.headerStyle.fontFamily = 'Roboto';
```

Comment:
Please make this change in global-style.js directly.

Reply:
Done Sir.